### PR TITLE
Multiple minor fixes to notebooks under tutorials directory

### PR DESCRIPTION
**Pull request by frreiss**
This PR groups together a number of minor fixes to the notebooks and Python files under the `tutorials` directory. Summary of changes:
* Fixed path issue that caused `CoNLL_2.ipynb` not to run
* Added one more cell to `CoNLL_2.ipynb` to generate the example DataFrame used in the PyData Montreal talk.
* Fixed a minor typo in a comment in `Market_Intelligence_Part1.ipynb`
* Edited the Markdown text of `Market_Intelligence_Part2.ipynb` based on edits from the Medium blog version
* Added the code-only notebooks, drawn directly from `Market_Intelligence_Part1.ipynb`, that were used in the PyData Montreal demo.
* Removed unnecessary cells from one of the Github gist notebooks for `Market_Intelligence_Part2.ipynb`
* Removed unused code from `market_intelligence.py` and synced some functions with their Github gist versions.

### CODE REVIEW

**Comment by frreiss** [comment]
@BryanCutler do you think you might have a chance to review these changes? I have some other changes queued up behind them.

File: tutorials/market/Market_Intelligence_Part2.ipynb [hunk 1]
```diff
@@ -250,7 +250,7 @@
    "cell_type": "markdown",
```
Comment (BryanCutler) [inline tutorials/market/Market_Intelligence_Part2.ipynb:250]: <p>I like this dependency parser description now, easier to understand</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

File: tutorials/market/PyData_Demo_1.ipynb [hunk 2]
```diff
@@ -0,0 +1,2455 @@
+{
```
Comment (BryanCutler) [inline tutorials/market/PyData_Demo_1.ipynb:1]: <p><code>copy()</code>  not necessary, reset_index will create a copy unless <code>inplace=True</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

File: tutorials/market/PyData_Demo_2.ipynb [hunk 3]
```diff
@@ -0,0 +1,336 @@
+{
```
Comment (BryanCutler) [inline tutorials/market/PyData_Demo_2.ipynb:1]: <p>same thing about reset_index and copy</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

**Review by BryanCutler — APPROVED** [review]
Looks great! I just had a suggestion about a possible redundant copy() in the notebooks. I'll go ahead and merge this, and up to you if you want to revise that or not.

File: tutorials/market/PyData_Demo_1.ipynb (hunk 2 above)
Comment (frreiss) [reply tutorials/market/PyData_Demo_1.ipynb:1]: <p>Good point, I'll drop that from the next version of this file.</p>

File: tutorials/market/PyData_Demo_2.ipynb (hunk 3 above)
Comment (frreiss) [reply tutorials/market/PyData_Demo_2.ipynb:1]: <p>Will fix this in the next commit.</p>

**Pull request merged**